### PR TITLE
run_exports and more tests

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by isuruf** [comment]
@pkgw, do you have any idea why libtool fails trying to find libxcb.la?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,45 +9,49 @@ source:
...
-    - m2w64-xz  # [win]
-    - pkg-config  # [not win]
-    - libtool  # [not win]
-    - xz  # [not win]
+    - posix                 # [win]
+    - m2w64-xz              # [win]
+    - pkg-config            # [not win]
+    - libtool               # [not win]
+    - automake              # [not win]
+    - xz                    # [not win]
```
Comment (bgruening) [inline recipe/meta.yaml]: xz should be in host, isn't it?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -9,45 +9,49 @@ source:
...
     - {{ compiler('c') }}
-    - pthread-stubs  # [linux]
-    - xcb-proto  # [linux]
-    - xorg-renderproto  # [linux]
-    - xorg-xproto  # [linux]
+    - pthread-stubs         # [linux]
+    - xcb-proto             # [linux]
```
Comment (bgruening) [inline recipe/meta.yaml]: Is it true that the x-libs are in build?

File: recipe/meta.yaml [hunk 3]
```diff
@@ -71,6 +75,12 @@ test:
     - test -f $PREFIX/lib/pkgconfig/cairo-quartz.pc   # [osx]
     - test -f $PREFIX/lib/pkgconfig/cairo-xlib.pc     # [linux]
 
+    # check include files
+    - test -f $PREFIX/include/cairo/cairo.h           # [unix]
+
+    # check that cairo was built with fontconfig support
+    - grep -q "CAIRO_HAS_FC_FONT 1" $PREFIX/include/cairo/cairo-features.h  # [unix]
```
Comment (jakirkham) [inline recipe/meta.yaml]: 👍

**Comment by jakirkham** [comment]
> do you have any idea why libtool fails trying to find libxcb.la?

Because it is not there. Chance are something else in the stack has a `.la` file that is referring to it.

File: recipe/meta.yaml (hunk 1 above)
Comment (isuruf) [reply recipe/meta.yaml]: No, this is just to unpack the archive.

**Comment by pkgw** [comment]
OK, the removal of the XCB Libtool `.la` files is breaking too much; I'll make a new build that puts them back. Hopefully that will fix this up.

**Comment by bgruening** [comment]
ping @isuruf its green now.

**Comment by isuruf** [comment]
Thanks for fixing conflicts. I'm not a maintainer of this feedstock. ping @conda-forge/cairo

**Pull request merged**